datamade/la-metro-councilmatic
# event and bill packet refactor

## Overview

This PR updates the way that we create Event and Bill packets.

## Testing Instructions

Assuming that you already have a local repo running

```bash
docker-compose run --rm -e DJANGO_MANAGEPY_MIGRATE=off app python manage.py compile_pdfs
```

Check out bill pages and event pages to see if the packets exist as expected. Not that it may take a moment for the packets to be processed by the pdf merger service.

Rerun

```bash
docker-compose run --rm -e DJANGO_MANAGEPY_MIGRATE=off app python manage.py compile_pdfs
```

No packets should be updated

If some time has passed since your last scrape, rerun a scrape and then rerun compile pdfs and any updated events and bills should update.
